commerce_coordinator/apps/ecommerce/constarnts.py

```diff
@@ -0,0 +1,2 @@
+""" Ecommerce Application Consonants """
```

Comment:
typo in the file name: `constants.py`

```suggestion
""" Ecommerce Application Constants """
```